src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+{
+	"global_executed_by": "Ausgeführt von {user_name}#{user_discriminator}",
+	"global_missing_permission": ":lock: Fehlende Rechte",
+	"global_missing_permission_bot": "Dem Bot fehlen Berechtigungen um dies durchzuführen! Bitte überprüfe die Rechte für den Bot.\n{missing}",
+	"global_user_not_found": "{emote_warn} Benutzer konnte nicht gefunden werden",
+	"global_channel_not_found": "{emote_warn} Channel konnte nicht gefunden werden",
+	"global_mod_action_log": "{action} by: {moderator_tag} ({moderator_id}) | Target: {target_tag} ({target_id}) | Grund: {reason}",
```

I'm not happy with how I translated "target" as "Ziel" quite yet. I tried finding similar meanings, e.g. "actioned user" but they don't really make sense in German quite yet. Improvements appreciated.

```suggestion
	"global_mod_action_log": "{action} von: {moderator_tag} ({moderator_id}) | Ziel: {target_tag} ({target_id}) | Grund: {reason}",
```